content/en/docs/pipelines/sdk/python-function-components.ipynb

```diff
@@ -53,14 +54,19 @@
    "outputs": [],
    "source": [
     "import kfp\n",
-    "import kfp.components as comp"
+    "from kfp.components import create_run_from_pipeline_func, InputPath, OutputPath"
```

Comment:
The original form is better 

Per Google [styleguide](https://google.github.io/styleguide/pyguide.html#22-imports):

> Use import statements for packages and modules only, not for individual classes or functions. Note that there is an explicit exemption for imports from the [typing module](https://google.github.io/styleguide/pyguide.html#typing-imports).

Reply:
Done